apps/site/src/views/Home/sections/Landing/Landing.tsx

```diff
@@ -7,6 +7,7 @@ function Landing() {
 	return (
 		<div className={styles.landing}>
 			<h1>ZotHacks 2023</h1>
```

Comment:
Perhaps more of a question for marketing/graphics, but now that we have the date, does it make sense to still include the year in the title?

Reply:
I would leave it in because other hackathons that use the year use it in the title. Actually, I was considering removing the year from the date.

Reply:
Sure, let's omit the year from the date.